### ISSUE TITLE
error

### ISSUE THREAD

**Issue by meredith2020** [jiawei-ren/BalancedMSE#23]
In this line of code “loss = F.cross_entropy(logits, torch.arange(pred.shape[0]))  ”, the input for 'logits' is [batch, batch], while the input for 'pred' is [batch, 1]. The dimensions of these two do not match. Are you sure they can be directly used to calculate the cross-entropy loss? My code throws an error at this point.
